config/metrics_endpoints.json

```diff
@@ -1,635 +1,1111 @@
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [],
+        "docker": []
+    },
+    {
+        "enabler": "Idra",
+        "catalogue": "",
+        "academy": "",
+        "readthedocs": "idra",
+        "helpdesk": "IDRA",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "OPSILab/Idra"
+        ],
+        "docker": [
+            "idraopendata/idra"
+        ]
+    },
+    {
+        "enabler": "Interface Designer",
+        "catalogue": "interface-designer",
+        "academy": "Interface Designer",
+        "readthedocs": "interface-designer",
+        "helpdesk": "UI Designer",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Adminotech/fiware-interface-designer"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "IoT Agent Node.JS Library",
+        "catalogue": "",
+        "academy": "",
+        "readthedocs": "iotagent-node-lib",
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/iotagent-node-lib"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "IoT Agent for JSON",
+        "catalogue": "",
+        "academy": "IDAS",
+        "readthedocs": "fiware-iotagent-json",
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/iotagent-json"
+        ],
+        "docker": [
+            "fiware/iotagent-json"
+        ]
+    },
+    {
+        "enabler": "IoT Agent for Lightweight M2M",
+        "catalogue": "",
+        "academy": "IDAS",
+        "readthedocs": "fiware-iotagent-lwm2m",
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/lightweightm2m-iotagent"
+        ],
+        "docker": [
+            "fiware/lightweightm2m-iotagent"
+        ]
+    },
+    {
+        "enabler": "IoT Agent for LoRaWan",
+        "catalogue": "",
+        "academy": "",
+        "readthedocs": "fiware-lorawan",
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Atos-Research-and-Innovation/IoTagent-LoRaWAN"
+        ],
+        "docker": [
+            "fiware/iotagent-lorawan"
+        ]
+    },
+    {
+        "enabler": "IoT Agent for OPC UA",
+        "catalogue": "",
+        "academy": "",
+        "readthedocs": "iotagent-opcua",
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Engineering-Research-and-Development/iotagent-opcua"
+        ],
+        "docker": [
+            "rdlabengpa/opcuaage"
+        ]
+    },
+    {
+        "enabler": "IoT Agent for SigFox",
```

Comment:
Sigfox

Reply:
Fixed in https://github.com/flopezag/fiware-tsc-dashboard/pull/38/commits/55ea9ac2cb5f6bb623a5b2897a9d1e5dede5c585